.gitignore

```diff
@@ -15,3 +15,4 @@
 \#*
 *.log
 *.png
+*.zip
```

Comment:
I would change this to `pw-trace.zip` because we should be aware of other zip files that get created unexpectedly.
Also is there any reason not to delete this file at the end of an `e2e` test?

Reply:
You're right.
I gonna improve and fix a place for this kind of files.

Those files are really helpful for CI debug. 
Playwright has a nice solution that we can record the test [traces](https://playwright.dev/docs/trace-viewer/) and play it as a "video".